pkg/ovs/ovsconfig/ovs_client_windows.go

```diff
@@ -0,0 +1,25 @@
+// +build windows
+
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ovsconfig
+
+import (
+	"github.com/vmware-tanzu/antrea/pkg/agent/util"
+)
+
+func (br *OVSBridge) SetInterfaceMTU(name string, MTU int) error {
+	return util.SetInterfaceMTU(name, MTU)
```

Comment:
Windows uses different mechanism to set MTU for internal port?

Reply:
Without this patch, windows follows the same logic of linux configuring MTU, but it fails judging from the result.